prboom2/src/dsda/console.c

```diff
@@ -213,6 +215,18 @@ static dboolean console_Exit(const char* args) {
   return true;
 }
 
+static dboolean console_CheatEnter(const char* args) {
+  char element[CONSOLE_ENTRY_SIZE];
+  char value[3];
```

Comment:
What happens if the string size of value is greater than 2? You can protect against this by using `%2s` in `sscanf` to restrict the length. It isn't used like this elsewhere because the code that fills the console buffer already restricts the total to `CONSOLE_ENTRY_SIZE`.